src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2909,6 +2909,67 @@ curl -X GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Limits for Plan
+
+To keep services stable for all clients, LiveChat limits common resources. The limits are different for each pricing plan.
```

Comment:
So this method compares the requester license's resources to limits for a plan they use in the request? E.g. Requester on business calls `check_product_limits_for_plan` for Team and it returns specifically what goes above limits for **their license**, right?

If the answer to the above is yes, then I'd suggest this description:

> Resource limits in the LiveChat product are different for each plan. This method compares your organization's current resources with a given plan and returns those which exceeded the called plan's limits.

Reply:
wow, magic. changed here https://github.com/livechat/livechat-public-docs/pull/1099/commits/0054d962d984ed70136756af92403d1a0dbe0e4c